Readme.txt

```diff
@@ -1,77 +1,138 @@
                                 MORSE RUNNER
                               Contest Simulator
                                   freeware
-
-                   Version 1.80 Field Day Contest Prototype
+                                  
+                        Version 1.82 - ARRL DX Contest
+            The third release of the Morse Runner Community Edition
 
                Copyright (C) 2004-2016 Alex Shovkoplyas, VE3NEA
                       http://www.dxatlas.com/MorseRunner/
 
-        Copyright (C) 2022 Morse Runner Community Edition Contributors
+        Copyright (C) 2022-2023 Morse Runner Community Edition Contributors
                    https://www.github.com/w7sst/MorseRunner/
 
 
 JOIN OUR COMMUNITY
   You are invited to join our community effort.
   For more information on the Morse Runner Community Edition project,
-  please visit https://github.com/w7sst/MorseRunner#readme.
+  please visit https://github.com/w7sst/MorseRunner#readme.  
+  Feedback can be left in our discussions area. 
```

Comment:
This is a minor point, but I see your editor is adding whitespace at the end of some lines.

Reply:
I'll have to watch for this. I'm using the editor within the GitHub webpage. Thanks for the tip!